### ISSUE TITLE
[P2][Sourcing][Accessibility][Pretest][Multi input autocomplete] If an autocomplete entry is selected with ENTER, focus is lost to the top of the page. 

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10814]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
If an autocomplete entry is selected with ENTER, focus is lost to the top of the page. 
Expectation : Keyboard focus should be placed back in the input field.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular: 15
core/platform: 0.43.14

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
1) Move to multi-input field in example https://ng-15-downport--fundamental-ngx.netlify.app/#/core/multi-input#includes
2) now search "App" ,apple gets displayed
3) Press down arrow to move to autocomplete list
3) press enter to select "Apple" from the list
4) token comes in the input field, at this moment focus is still at list item "Apple"
5) Now press Tab key and see that focus leaves the multi input control

[Uploading Screen Recording 2023-10-26 at 4.21.54 PM.mov.zip…]()

**Comment by droshev** [comment]
@I543348 if you are on step 6 and pressing Tab, it is expected the focus go outside of the multi input menu => there is no issue here.


Nevertheless what we need to change in the scenario is:
1. Move to multi-input field in example https://ng-15-downport--fundamental-ngx.netlify.app/#/core/multi-input#includes
2. now search "App" ,apple gets displayed
3. Press down arrow to move to autocomplete list
4. press enter to select "Apple" from the list
The expected behaviour is that the token `Apple` is added to the input, the written text gets deleted and the focus should go back in the input next to the token.

**Comment by I543348** [comment]
> @I543348 if you are on step 6 and pressing Tab, it is expected the focus go outside of the multi input menu => there is no issue here.
> 
> Nevertheless what we need to change in the scenario is(after step 4):
> 
> 1. Move to multi-input field in example https://ng-15-downport--fundamental-ngx.netlify.app/#/core/multi-input#includes
> 2. now search "App" ,apple gets displayed
> 3. Press down arrow to move to autocomplete list
> 4. press **ENTER** to select "Apple" from the list
>    The expected behaviour is that the token `Apple` is added to the input, the written text gets deleted and the focus should go back in the input next to the token, and the list gets closed. ([example](https://ui5.sap.com/#/entity/sap.m.MultiComboBox/sample/sap.m.sample.MultiComboBox))
> 
> Another scenario where we don't need to change the behaviour:
> 
> 1. Move to multi-input field in example https://ng-15-downport--fundamental-ngx.netlify.app/#/core/multi-input#includes
> 2. now search "App" ,apple gets displayed
> 3. Press down arrow to move to autocomplete list
> 4. press **SPACE** to select "Apple" from the list
>    The expected behaviour is a new token is created in the input, the focus stays on the item (as it is working right now => no changes needed)

Hi @droshev , We would be fine with both the approaches you mentioned, they look good. Thanks.
